pulp_rpm/app/viewsets.py

```diff
@@ -243,3 +245,13 @@ def create(self, request):
             }
         )
         return OperationPostponedResponse(result, request)
+
+
+class RpmDistributionViewSet(BaseDistributionViewSet):
```

Comment:
since `remote` field was removed from BaseDistribution, we need to add it in a subclassed one.
@pavelpicka have you tested distribution of a content that was sycned with on_demand or streamed policy?

Reply:
https://pulp.plan.io/issues/4817

Reply:
Tested with "on_demand", "streamed" and "immediate" policy on remote and after test possibility to install packages via dnf and all three went ok.

Reply:
I will rebase this PR https://github.com/pulp/pulp_rpm/pull/1339 and we will have those tests to consume content lazy synced.

Reply:
pull though cache covers the use case when plugins do not have metadata,basically as of now it is a maven plugin.
there was some confusion around terminology pass-through and pull-through cache.
so we don't need to add remote in the rpm plugin case.